Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocTextValue.css

```diff
@@ -27,11 +34,7 @@ span.bocTextValue .content
   left: 0;
   right: 6px;
```

Comment:
see BocMultilineTextValue. please also review other remarks and apply here as needed.